app/jobs/reports/identity_reuse_report.rb

```diff
@@ -0,0 +1,51 @@
+require 'identity/hostdata'
+
+module Reports
+  class IdentityReuseReport < BaseReport
+    REPORT_NAME = 'credential-reuse-report'.freeze
+
+    include GoodJob::ActiveJobExtensions::Concurrency
+
+    good_job_control_concurrency_with(
+      total_limit: 1,
+      key: -> { "#{REPORT_NAME}-#{arguments.first}" },
+    )
+
+    def perform(_date)
+      report = {
+        total_ial1_identity_counts: ial1_identity_counts,
+        total_unique_ial1_identity_counts: unique_ial1_identity_counts,
+        total_ial2_identity_counts: ial2_identity_counts,
+        total_unique_ial2_identity_counts: unique_ial2_identity_counts,
```

Comment:
what if we added a query with distributions? ex # of users with 2 identities, # of users with 3, etc... a query could look like:

```sql
SELECT
  COUNT(*) AS num_users
, subq.num_identities AS num_identities
FROM
(
  SELECT
    COUNT(*) AS num_identities
  , identities.user_id
  FROM identities
  WHERE
      identities.ial = 1
  AND identities.created_at < '2022-01-01'
  GROUP BY identities.user_id
) subq
GROUP BY subq.num_identities
ORDER BY subq.num_identities DESC
```

Reply:
It's superfluous as far as total stats are concerned since one user reused twice is the same as two users each reused once.